backend/src/main/java/com/springBoot/fruits_ecommerce/services/JwtService.java

```diff
@@ -0,0 +1,80 @@
+package com.springBoot.fruits_ecommerce.services;
+ 
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.crypto.SecretKey;
+
+import org.springframework.stereotype.Service;
+
+import com.springBoot.fruits_ecommerce.models.Role;
+import com.springBoot.fruits_ecommerce.models.User;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+
+@Service
+public class JwtService {
+    private final String secret="12df32e69496a167c9d6196abec4bd4a25858e8da5fa4fe150ac9e03431ca36b";
```

Comment:
Let's rename secret to SECRET as its static and constant, and move its value to be environment variable instead of adding directly to source code